clang/lib/DPCT/APINamesMisc.inc

```diff
@@ -121,3 +121,33 @@ CONDITIONAL_FACTORY_ENTRY(
                  CALL(MapNames::getDpctNamespace() +
                           "dev_mgr::instance().get_device",
                       ARG(1)))))))
+
+CONDITIONAL_FACTORY_ENTRY(
```

Comment:
You can use ASSIGNABLE_FACTORY here.

Reply:
Replace by REMOVE_API_FACTORY_ENTRY_WITH_MSG rewriter, help review again.